homeassistant/components/homeassistant_green/strings.json

```diff
@@ -0,0 +1,28 @@
+{
+  "options": {
+    "step": {
+      "hardware_settings": {
+        "title": "Configure hardware settings",
+        "data": {
+          "disk_led": "Green: activity LED",
+          "power_led": "White: power LED",
+          "user_led": "Yellow: system health LED"
+        }
+      },
+      "reboot_menu": {
+        "title": "Reboot required",
+        "description": "The settings have changed, but the new settings will not take effect until the system is rebooted",
+        "menu_options": {
+          "reboot_later": "Reboot manually later",
+          "reboot_now": "Reboot now"
+        }
+      }
```

Comment:
Don't we always use restart instead of reboot?

Reply:
No:
![image](https://github.com/home-assistant/core/assets/14281572/2f4cfc42-81ed-4075-a189-9744f754515a)

Reply:
I love how that screenshot shows 1 reboot and 4 restarts 😂

Reply:
No matter what, we refer to a full system restart as "reboot system", so we should use the same language in the options flow.